### PR TITLE
Add timeouts to tm start/stop test

### PR DESCRIPTION
**Pull request by portante**

### CODE REVIEW

File: agent/util-scripts/test-bin/test-start-stop-tool-meister [hunk 1]
```diff
@@ -56,17 +56,21 @@ printf -- "Exit code: %d\n" ${?} >> ${_ptms_output}
...
 
+function _timeout {
+    timeout --kill-after=10 --signal TERM 20 $*
+}
+
 # Now execute the success case.
 source ${_script_path}/common-tm-cleanup
 
-_PBENCH_TOOL_MEISTER_LOG_LEVEL="debug" _PBENCH_TOOL_DATA_SINK_LOG_LEVEL="debug" _PBENCH_TOOL_MEISTER_START_LOG_LEVEL="debug" PBENCH_REDIS_SERVER="localhost" PBENCH_TOOL_DATA_SINK="localhost:8080;localhost:8080" pbench-tool-meister-start --sysinfo="none" ${group}
+_PBENCH_TOOL_MEISTER_LOG_LEVEL="debug" _PBENCH_TOOL_DATA_SINK_LOG_LEVEL="debug" _PBENCH_TOOL_MEISTER_START_LOG_LEVEL="debug" PBENCH_REDIS_SERVER="localhost" PBENCH_TOOL_DATA_SINK="localhost:8080;localhost:8080" _timeout pbench-tool-meister-start --sysinfo="none" ${group}
```
Comment (dbutenhof) [inline agent/util-scripts/test-bin/test-start-stop-tool-meister:66]: Not related to this PR, but why we we specify just one server address for Redis, but identical bind and connection addresses for TDS? Or is that just an additional test case for the `BaseServer.__init__` logic?

File: agent/util-scripts/test-bin/test-start-stop-tool-meister [hunk 2]
```diff
@@ -56,17 +56,21 @@ printf -- "Exit code: %d\n" ${?} >> ${_ptms_output}
 cat -n ${_ptms_output}
 rm ${_ptms_output}
 
+function _timeout {
+    timeout --kill-after=10 --signal TERM 20 $*
```
Comment (dbutenhof) [inline agent/util-scripts/test-bin/test-start-stop-tool-meister:60]: It's not necessary, but you're usually so careful about using `${*}` syntax, this stands out.

**Review by dbutenhof — APPROVED** [review]
Unimportant question/comment which, if ignored, I'll just resolve. 😆

File: agent/util-scripts/test-bin/test-start-stop-tool-meister (hunk 1 above)
Comment (portante) [reply agent/util-scripts/test-bin/test-start-stop-tool-meister:66]: Good question.  I think, but not entirely sure now, that this was how we got things to force `localhost` by use a non-`localhost` name in the tests.

Comment (webbnh) [inline agent/util-scripts/test-bin/test-start-stop-tool-meister:66]: This line would be much more comprehensible if it were wrapped:
```
_PBENCH_TOOL_MEISTER_LOG_LEVEL="debug" \
    _PBENCH_TOOL_DATA_SINK_LOG_LEVEL="debug" \
    _PBENCH_TOOL_MEISTER_START_LOG_LEVEL="debug" \
    PBENCH_REDIS_SERVER="localhost" \
    PBENCH_TOOL_DATA_SINK="localhost:8080;localhost:8080" \
    _timeout pbench-tool-meister-start --sysinfo="none" ${group}
```
Ditto, line 73.

**Review by webbnh — APPROVED** [review]
👍

**Pull request merged**